minio/resource_minio_ilm_policy_test.go

```diff
@@ -90,6 +90,27 @@ func TestAccILMPolicy_filterTags(t *testing.T) {
 	})
 }
 
+func TestAccILMPolicy_expireNoncurrentVersion(t *testing.T) {
+	var lifecycleConfig lifecycle.Configuration
+	name := fmt.Sprintf("test-ilm-rule4-%d", acctest.RandInt())
+	resourceName := "minio_ilm_policy.rule4"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { testAccPreCheck(t) },
+		ProviderFactories: testAccProviders,
+		CheckDestroy:      testAccCheckMinioS3BucketDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccMinioILMPolicyExpireNoncurrentVersion(name),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckMinioILMPolicyExists(resourceName, &lifecycleConfig),
+					testAccCheckMinioLifecycleConfigurationValid(&lifecycleConfig),
```

Comment:
Can we add a check for the specific property value here to verify that it assigns the right number of days? Once we've got that I think this is good to go!

Reply:
Something like that you mean?
```suggestion					
					testAccCheckMinioILMPolicyExists(resourceName, &lifecycleConfig),
					testAccCheckMinioLifecycleConfigurationValid(&lifecycleConfig),
					resource.TestCheckResourceAttr(
						resourceName, "rule.0.noncurrent_version_expiration_days", "5"),
```

Reply:
Yeah that would work great!

Reply:
Done